spqlios/cplx/cplx_fft_ref.c

```diff
@@ -69,8 +69,8 @@ void cplx_fft16_ref(void* data, const void* omega) {
   citwiddle(d[14], d[15], om[7]);
 }
 
-double cos_2pix(double x) { return cos(2 * M_PI * x); }
-double sin_2pix(double x) { return sin(2 * M_PI * x); }
+double cos_2pix(double x) { return m_accurate_cos(2 * M_PI * x); }
+double sin_2pix(double x) { return m_accurate_sin(2 * M_PI * x); }
```

Comment:
`M_PI` is from libm

Reply:
for now, we can stick to using libm: 
some unittests spotted a small difference in accuracy between linux and windows complex fft, but after checking, it does not seem related to the accuracy of sin and cosinus as I first suspected. maybe some rounding?

I leave the internal taylor-series based version of sinus and cosinus, in case later, we would need to come back and get rid of libm completely, but for now, it is not used and the macro just defines `m_accurate_cos` as `cos`.